jenkins/ci.suse.de/pipelines/openstack-ardana-bootstrap.Jenkinsfile

```diff
@@ -87,6 +87,23 @@ pipeline {
       }
     }
 
+    stage('ansible linting') {
+      steps {
+        sh '''
+          set -x
+
+          # Make a copy of the ansible virtualenv
```

Comment:
There are several things incorrect about this approach:
- you don't need to create a copy of the virtualenv, just manually install ansible-lint to the existing one on the Jenkins node (however, you probably won't need this either - see my next point)
- the linter is supposed to run over the `/var/lib/ardana/scratch/ansible/next/ardana/ansible` folder set up on the deployer, right ? so why would you run it here, on the Jenkins agent node ?

Reply:
Making a copy of the virtualenv prevents us from polluting the test environment. Only the cloud installer should install libraries into /opt/ansible, if we manually install ansible-lint there as well, we might accidentally change important dependency versions (including potentially changing ansible versions).

Reply:
I thought all pipeline stages ran on the agent node, are you suggesting that we move the linting from a pipeline stage to a deployer script?